cmd/dhcpserver/list.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 IBM Corp
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dhcpserver
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+
+	"github.com/ppc64le-cloud/pvsadm/pkg"
+	"github.com/ppc64le-cloud/pvsadm/pkg/client"
+	"github.com/ppc64le-cloud/pvsadm/pkg/utils"
+)
+
+var listCmd = &cobra.Command{
+	Use:   "list",
+	Short: "Get PowerVS DHCP servers",
+	Long:  `Get PowerVS DHCP servers`,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if pkg.Options.InstanceID == "" && pkg.Options.InstanceName == "" {
+			return fmt.Errorf("--instance-id or --instance-name required")
```

Comment:
I could not find declaration of `--instance-name` flag

Reply:
Its there in the parent command

Reply:
https://github.com/ppc64le-cloud/pvsadm/blob/65f27a949222108931fe8df8a0d478d8b5c9a06b/cmd/dhcpserver/cmd.go#L48 I am seeing `instance-id` only here

Could not find in its parent here too cmd/root.go 

Or if it is declared somewhere else, can you please point me there?

Reply:
removed this check as well.